topics/epigenetics/tutorials/methylation-seq/tutorial.md

```diff
@@ -69,7 +69,7 @@ The data we use in this tutorial is available at [Zenodo](https://zenodo.org/rec
 > We will map now the imported dataset against a reference genome.
 >
 > 1. **Galaxy** {% icon tool %}: Search for the tool ```bwameth```
-> 2. **bwameth** {% icon tool %}: Select for the option ```Select a genome reference from your history or a built-in index?``` ```Use a built-in idex``` and here the human ```hg38``` genome.
+> 2. **bwameth** {% icon tool %}: Select for the option ```Select a genome reference from your history or a built-in index?``` ```Use a built-in index``` and here the human ```hg38``` genome.
 > 3. **bwameth** {% icon tool %}: Choose for the option ```Is this library mate-paired?``` ```Paired-end``` and use the two imported datasets as an input.
```

Comment:
Are all these steps part of the same step to once `bwameth` or do you need to run `bwameth` several times?
In the first case, it may be better (easier to understand) to do something like:

```
> 1. **bwameth** {% icon tool %} with
>     - "Select a genome reference from your history or a built-in index?" to `Use a built-in index`
>     - "Select a reference genome" to `hg38` genome
>     - "Is this library mate-paired?" to `Paired-end`
>     - "First read in pair" and "Second read in pair" to the two imported datasets as an input.
>    
>      Please notice that depending on your system this computation can take some time. If you want to skip this, we provide for you a precomputed alignment: Import ```aligned_subset.bam``` to your history.
>
 {: .hands_on}
```

What do you think?
We used this format in the metagenomics tutorials. I found it better during the workshop to find which parameters changing and with which values.